actix-cors/src/lib.rs

```diff
@@ -1,9 +1,11 @@
-#![allow(clippy::borrow_interior_mutable_const, clippy::type_complexity)]
```

Comment:
Do not remove this in this PR.

Reply:
I think they aren't actually used anymore (or at least local clippy was not complaining). Are you asking to generally keep this annotation, or to submit a separate PR in order to drop it?
(I've added it back and grouped with other annotations for now)

Reply:
I'm in favor of removing these if they're not needed. Preferring more locally scoped exception if they are needed.

Reply:
Ack, split that one to https://github.com/actix/actix-extras/pull/75.

Reply:
Nevermind, I was wrong, my local clippy tricked me somehow. Forcing a check from scratch, I see that those lints are triggered all over the module. I think it makes sense to keep this as is.